server/src/main/java/com/lunchtime/repository/FeedbackRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.lunchtime.repository;
+
+import com.lunchtime.models.Feedback;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import java.util.List;
+
+
+@Repository
+public interface FeedbackRepository extends JpaRepository<Feedback, Long> {
+    List<Feedback> findByRestId_Id(Long id);
```

Comment:
Sorry, it seems that I don't understand what this method does. Could you explain, please?

Reply:
JPA magic 
https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#jpa.query-methods.query-creation
5.3.2. Query Creation

Reply:
My question wasn't about query creation but about **findByRestId_Id(Long id)** functionality :) What is second id referred to?

Reply:
(_Id) you speak about this?

Reply:
Yes, I do.